src/Main.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import HornedBeast from './HornedBeast.js';
+import Form from 'react-bootstrap/Form';
+import data from './data.json';
```

Comment:
Great work on this lab!

- This can work here, but it may help to move/import the data into App (lab 2 requirements) and pass the data to the components as props 👍